src/main/java/org/opensrp/util/constants/EventDataExportConstants.java

```diff
@@ -0,0 +1,26 @@
+package org.opensrp.util.constants;
+
+public class EventDataExportConstants {
+
+	public static final String SETTINGS_CONFIGURATION_EVENT_TYPE_LOOKS_GOOD = "event_type_looks_good";
+
+	public static final String SETTINGS_CONFIGURATION_EVENT_TYPE_FLAG_PROBLEM = "event_type_flag_problem";
+
+	public static final String SETTINGS_CONFIGURATION_EVENT_TYPE_FIX_PROBLEM = "event_type_fix_problem";
+
+	public static final String SETTINGS_CONFIGURATION_EVENT_TYPE_RECORDS_GPS = "event_type_records_gps";
+
+	public static final String SETTINGS_CONFIGURATION_EVENT_TYPE_SERVICE_POINT_CHECK = "event_type_service_point_check";
+
+	public static final String EVENT_LOOKS_GOOD = "Looks good";
+
+	public static final String EVENT_FLAG_PROBLEM = "Flag problem";
+
+	public static final String EVENT_FIX_PROBLEM = "Fix problem";
+
+	public static final String EVENT_RECORDS_GPS = "Records GPS";
+
+	public static final String EVENT_SERVICE_POINT_CHECK = "Service point check";
```

Comment:
Move this configuration to settings, so that we could add new events without code changes

Reply:
Okay